images/linux/scripts/tests/DotnetSDK.Tests.ps1

```diff
@@ -28,4 +28,15 @@ Describe "Dotnet" {
             }
         }
     }
+
+    $dotnetTools = (Get-ToolsetContent).dotnet.tools
+
+    foreach ($dotnetTool in $dotnetTools) {
+        Context "Dotnet tool $dotnetTool" {
+            It "dotnet tool '$dotnetTool' is available" {
+                
+                $dotnetTool
```

Comment:
We have to use `Should` and assert that we should to test.

Reply:
The reason for this being empty is that if you invoke a dotnet tool that doens't exist it will throw an exception and the test will fail. I guess that's a test wide and generic enough. Otherwise one would have to test for some result and that will vary from tool to tool...

Reply:
If we want to test an exception we should test the exception. Otherwise one would have to test for some result and that will vary from tool to tool - Definitely, we should test every tool for basic cases.

Look at `"npm":` section -  https://github.com/actions/virtual-environments/blob/3a13e887784a5348840e102d3ce617cc8eb2fd2a/images/win/toolsets/toolset-2016.json

Reply:
Specific test for each dotnet tool added.